Readme.md

```diff
@@ -0,0 +1,30 @@
+# flow-coverage-action
+
+A github action to lint for flow coverage! This allows you to have much more confidence while using a type system that has escape hatches :)
+
+## Using the action
+
+```
+	steps:
+    - name: flow coverage
+      uses: Khan/flow-coverage-action@main
+      with:
+        flow-bin: ./node_modules/.bin/flow
+        check-run-subtitle: 'React Native Folder'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
+## Using the cli
```

Comment:
Do we want the cli in this repo or in with our `git actions` tooling, which has a lot of overlap with this, but is currently in the olc repo. Maybe that needs to be moved out of there and brought instead in with these repos, since it pretty much just calls these?